crypto/curve25519/curve25519.c

```diff
@@ -1069,12 +1075,12 @@ static void ge_p2_dbl(ge_p1p1 *r, const ge_p2 *p) {
   fe_sq(r->X, p->X);
   fe_sq(r->Z, p->Y);
   fe_sq2(r->T, p->Z);
-  fe_add(r->Y, p->X, p->Y);
+  GFp_fe_add(r->Y, p->X, p->Y);
   fe_sq(t0, r->Y);
-  fe_add(r->Y, r->Z, r->X);
-  fe_sub(r->Z, r->Z, r->X);
-  fe_sub(r->X, t0, r->Y);
-  fe_sub(r->T, r->T, r->Z);
+  GFp_fe_add(r->Y, r->Z, r->X);
+  GFp_fe_sub(r->Z, r->Z, r->X);
+  GFp_fe_sub(r->X, t0, r->Y);
+  GFp_fe_sub(r->T, r->T, r->Z);
```

Comment:
This is a really performance-sensitive function. Does this change the code generation or performance in any substantial way?

Maybe a more conservative change would be to have  `GFp_fe_sub()` and `GFp_fe_add()` be `extern` wrappers around a `static` `fe_sub` and `fe_add`. Then we don't have to worry about potential performance implications as much (especially on ARM). We could then clean it up if/when we have a good story for verifying performance on non-X86_64 hardware. WDYT?